src/Actions/Actions.ts

```diff
@@ -62,21 +64,25 @@ export default class Actions {
 
           this.config.statsDb.updateClaimed(from, cost);
 
-          return txRequest.isClaimed;
+          if (txRequest.isClaimed) {
+            return ClaimStatus.SUCCESS;
+          }
         }
 
-        return false;
+        return ClaimStatus.SUCCESS;
```

Comment:
I think it should be failed  here instead of success, because `isTransactionStatusSuccessful` is `false` at this point.

Reply:
You're correct, missed this.

Reply:
Agree. I had the same comment. Otherwise approved